run_unit_tests.sh

```diff
@@ -40,7 +40,6 @@ which airflow > /dev/null || python setup.py develop
 
 echo "Initializing the DB"
 yes | airflow resetdb
-airflow initdb
```

Comment:
removing this because `airflow resetdb` calls `airflow initdb`, so it is no longer needed